.github/workflows/selfdrive_tests.yaml

```diff
@@ -21,7 +21,7 @@ env:
 
   RUN: docker run --shm-size 2G -v $PWD:/tmp/openpilot -w /tmp/openpilot -e CI=1 -e PRE_COMMIT_HOME=/tmp/pre-commit -e PYTHONWARNINGS=error -e FILEREADER_CACHE=1 -e PYTHONPATH=/tmp/openpilot -e NUM_JOBS -e JOB_ID -e GITHUB_ACTION -e GITHUB_REF -e GITHUB_HEAD_REF -e GITHUB_SHA -e GITHUB_REPOSITORY -e GITHUB_RUN_ID -v $GITHUB_WORKSPACE/.ci_cache/pre-commit:/tmp/pre-commit -v $GITHUB_WORKSPACE/.ci_cache/scons_cache:/tmp/scons_cache -v $GITHUB_WORKSPACE/.ci_cache/comma_download_cache:/tmp/comma_download_cache -v $GITHUB_WORKSPACE/.ci_cache/openpilot_cache:/tmp/openpilot_cache $BASE_IMAGE /bin/bash -c
 
-  PYTEST: pytest --rootdir=openpilot --continue-on-collection-errors --cov --cov-report=xml --cov-append --durations=0 --durations-min=5 --hypothesis-seed 0 -n logical
```

Comment:
why was `--rootdir=openpilot` removed? wasn't the goal to discover and run unit tests inside just the openpilot directory? 

Tests are passing since it's just using the directories found in root and not in `/openpilot`

Reply:
it changes the local workflow, so i'd have to add "--rootdir" when running locally. this achieves mostly the same thing (discover all tests), but has no effect on the workflow. 

eventually, I think we'll end up moving `selfdrive/` and friends to actually be inside `openpilot/`, and this will all just work.

Reply:
ah gotcha, yeah it's a shame `--rootdir` doesn't work when setting it in pyproject.toml